### DISCUSSION TITLE
I need to get my account back

### DISCUSSION THREAD

**Discussion by guoweiTang** [npm/feedback#715] [Website] [answered]
I lost my npm 2fa code and recovery code, so what can i do to get my accout?
My username is ice.tang.

**Comment by MylesBorins** [comment, marked as answer]
Please [open a ticket with support](https://www.npmjs.com/support)

**Reply by guoweiTang** [reply]
But when i submit a request on the target page, an 403 error is returned.

**details:**
url: https://www.npmjs.com/support
method: post
data: {
  inquire: "account"
  security-inquire: 
  name: "ice.tang"
  email: "1214475704@qq.com"
  subject: "I want to get my account"
  message: "I lost my npm 2fa code and recovery code, so what can i do to get my accout?"
  csrftoken: "QD5C6lsYA5iYouysu-EUDiQcjBk2LO5BFwCvgKf0qbJ"
}
